src/outcome.rs

```diff
@@ -32,6 +32,8 @@ pub enum ClaimValue {
     Text { value: String },
     /// A date claim in the format `[year]-[month]-[day]`.
     Date { value: String },
+    /// MapArray
+    StatusList { value: HashMap<String, String> },
```

Comment:
Can this type just be a generic `Map` rather than `StatusList`?

Reply:
Per my comment on the other PR, I think `StatusList` may make more sense for now since it's currently used for specific parsing of the status list structure. I don't have a strong opinion here & I think it's non-blocking though!